_episodes/05-checklists.md

```diff
@@ -62,4 +63,9 @@ Here is a list of some candidate checklists:
 >
 {: .discussion}
 
+> ## The limits of a checklist
+> Are all software that ticked all the boxes of a software checklist high quality software?
+>
+{: .discussion}
+
```

Comment:
I am curious if this is just something that I find important or that others agree that this is an important note to add to this lesson?

Reply:
Interesting discussion point for a workshop!